src/parsley/utils.js

```diff
@@ -93,6 +101,22 @@ define('parsley/utils', function () {
         .toLowerCase();
     },
 
+    objectCreate: Object.create || (function() {
```

Comment:
Found out it might be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create ;)